service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -80,12 +79,14 @@ public CaseDataExtractionService(DateFormatterService dateFormatterService,
     public Map<String, Object> getStandardOrderDirectionData(CaseData caseData) throws IOException {
         ImmutableMap.Builder data = ImmutableMap.<String, Object>builder();
 
-        if (isNotEmpty(caseData.getStandardDirectionOrder())) {
-            data.put("judgeAndLegalAdvisor", prepareJudgeAndLegalAdvisor(
-                caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor()));
-        } else {
-            data.put("judgeAndLegalAdvisor", prepareJudgeAndLegalAdvisor(JudgeAndLegalAdvisor.builder().build()));
-        }
+        JudgeAndLegalAdvisor judgeAndLegalAdvisor = isNotEmpty(caseData.getStandardDirectionOrder())
+            ? caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor() : caseData.getJudgeAndLegalAdvisor();
+
+        data.put("judgeTitleAndName", defaultIfBlank(JudgeAndLegalAdvisorHelper.formatJudgeTitleAndName(
+            judgeAndLegalAdvisor), EMPTY_PLACEHOLDER));
+        //Legal advisor will be tied to magistrate in following story - this + tests will need updating then
```

Comment:
actually this comment is incorrect now after discussions with BA - legal advisor now optional and not conditional on magistrate. following story still addresses this though 👍